examples/guides/services/healthcare-service/tests/clemency-hospital-service-test.bal

```diff
@@ -0,0 +1,547 @@
+                        "hospital":"clemency medical center",
+                        "appointmentDate":"2019-08-30"
+                    },
+                    {
+                        "statusCode":200,
+                        "appoinmentNumber":true,
+                        "doctorAvailibility":true,
+                        "doctorFee":true
+                    } 
+                ],
+                [
+                   {  
+                        "patient":{  
+                            "name":"D Serasinghe",
+                            "dob":"1983-12-03",
+                            "ssn":"777-29-585",
+                            "address":"Colombo SL",
+                            "phone":"5578521755",
+                            "email":"dserasinghe@hotmail.com"
+                        },
+                        "doctor":"T Uyanage",
+                        "hospital":"clemency medical center",
+                        "appointmentDate":"2019-12-31"
+                    },
+                    {
+                        "statusCode":400,
+                        "responseMessage":"Doctor T Uyanage is not available in clemency medical center"
+                    }
+                ],
+                [
+                   {  
+                        "patient":{  
+                            "name":"Little John",
+                            "dob":"2018-04-29",
+                            "ssn":"",
+                            "address":"California",
+                            "phone":"",
+                            "email":""
+                        },
+                        "doctor":"cailen cooper",
+                        "hospital":"clemency medical center",
+                        "appointmentDate":"2019-12-02"
+                    },
+                    {
+                        "statusCode":200,
+                        "appoinmentNumber":true,
+                        "doctorAvailibility":true,
+                        "doctorFee":true
+                    } 
+                ]
+            ]; 
+        }
+
+# Description: This test scenario verifies if details of the reserved appoinment can be retrived. 
+# TC004 - Verify if appoinment details can be retrieved successfully by providing a valid appointment number.
+# TC005 - Verify if an error occurs by providing an invalid appointment number.  
+#
+# + dataset - dataset Parameter Description
+@test:Config
+{
+    dataProvider: "testGetAppointmentClemencyDataProvider",
+    dependsOn: ["testReserveAppointment"]
+}
+
+
+function testGetAppointmentClemency(json dataset)
+{
+    var expectedAppointmentNumber = dataset.appointmentNumber;
+    var expectedDoctorName = dataset.doctorName;
+    var expectedAppointmentDate = dataset.appointmentDate;
+
+    var response = clientEPclemency->get("/appointments/"+expectedAppointmentNumber.toString());
+    if(response is http:Response)
+    {
+        if(expectedAppointmentNumber == 20000000)
```

Comment:
Magic numbers? Shall we try to improve the readability?

Reply:
Corrected.